firebase/README.md

```diff
@@ -10,24 +10,66 @@ You will need to set up a new firebase project and enable the following features
+
+(If successful, `npm start` will now get you to an error page within the app)
+
+## Billing
+
+The Firebase functions we use make API calls to slack. As this is an external network request, you will be required to enable billing on your firebase account and change your plan to the Blaze pay-as-you-go pricing plan (or better).
+
+To do this for ScottLogic, you will need to gain access to ScottLogic's Billing account. See Chris Price or Colin Eberhardt for details.
+
 ### Storage
 
-Storage rules will be automatically configured when you deploy the application to firebase. However, you will also need to configure the lifecycle and CORS settings. To do so you must first install `gsutil` from [here](https://cloud.google.com/storage/docs/gsutil_install). Then, navigate to this directory in your terminal before running the following 2 commands:
+Storage rules will be automatically configured when you deploy the application to firebase (see below).
+
+However, you will also need to configure the lifecycle and CORS settings.
+
+To do so you must first install `gsutil` from [here](https://cloud.google.com/storage/docs/gsutil_install). Then, navigate to this directory in your terminal before running the following 2 commands:
+
+Now you need to set up gsutil to use your account
+
+Run 
+```
+$ gsutil config
+$ gsutil config
+```
+
+And follow the instructions for logging in with your Google account and setting up a Cloud Platform.
+
+Now add a new bucket for your hosting. This will be a s
 
 ```shell
-$ gsutil lifecycle set lifecycle.json gs://snackchat
-$ gsutil cors set cors.json gs://<PROJECT_ID>.appspot.com
+{HONESTY_DIR}\firebase $ gsutil lifecycle set lifecycle.json gs://<YOUR_SNACKCHAT_LOCATION>
+{HONESTY_DIR}\firebase $ gsutil cors set cors.json gs://<PROJECT_ID>.appspot.com
 ```
 
 Note: The last command configures CORS to allow requests from any URL. This is useful for debugging purposes and running the front end locally but may not be desirable in production. You can modify [`cors.json`](cors.json) to allow specific domains if needed.
 
+You will also need to update storage.rules to reference your buckets
+
+```
+  ...
+  match /b/{SNACKCHAT_DIR}/o {
+  ...
+  match /b/{YOUR_PROJECT_ID}.appspot.com/o {
+  ...
+```
+
 Storage is currently used to save the training data, evaluation data and snackchats (deleted after 24 hours)
 
 ### Authentication
 
-For the purpose of authenticating the kiosk to make calls to our firebase functions, you should create an Email/Password user account. Note down the UID of this user for configuring functions in the next step.
+For the purpose of authenticating the kiosk to make calls to our firebase functions, you should create an Email/Password user account.
+
+Note down the UID of this user for configuring functions in the next step.
```

Comment:
Did this not change with the auth changes?

Reply:
Yep. Removing appropriate lines.